CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/perspectives/PerspectiveModel.java

```diff
@@ -33,8 +33,7 @@ public PerspectiveModel() {
     }
 
     public PerspectiveModel(final PerspectiveModel other) {
-        perspectives = new ArrayList<>();
-        other.perspectives.stream().forEach(perspectives::add);
+        perspectives = (ArrayList<Perspective>) other.perspectives.stream().toList();
```

Comment:
Looking at the class, I suspect it would be better to take the case out here and instead change `perspectives` to be declared as simply a `List` rather than specifically an `ArrayList`

Reply:
I looked into this. `perspectives` needs to be serialized, which `ArrayList` is but a regular `List` is not. So the cast is necessary, unless I've missed something.

Reply:
Gotcha. Yep forgot about that other sonar rule